### PR TITLE
Group features in categories

### LINKED ISSUE #66: Organize features in categories
Features should be organized into categories to help discovery

### PR DESCRIPTION
**Pull request by alvarosanchez**
Fixes #66.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -20,9 +20,11 @@ To run the tests use `./gradlew check`.
 
 ### Running the CLI
 
-To generate a project execute `./gradlew run --args=""` where the args are what would be after `mn` using the CLI. For example: `./gradlew run --args="create-app temp"`. 
+To generate a project execute `./gradlew micronaut-cli:run --args=""` where the args are what would be after `mn` using the CLI. For example: `./gradlew micronaut-cli:run --args="create-app temp"`. 
 
-If you want to test output colors or the interactive shell, the jar must be executed. Run `./gradlew assemble` and then `java -jar starter-core/build/libs/starter-core-1.0.0.BUILD-SNAPSHOT.jar create-app temp`
+If you want to test output colors or the interactive shell, the jar must be executed. Run `./gradlew micronaut-cli:shadowJar` and then `java -jar starter-cli/build/libs/micronaut-cli-2.0.0.BUILD-SNAPSHOT-all.jar create-app temp`
+
+Note that interactive mode (ie: running without arguments) doesn't work when running the JAR [due to this issue](https://github.com/micronaut-projects/micronaut-starter/issues/130).
```
Comment (jameskleeh) [inline CONTRIBUTING.md:27]: This seems to be resolved?

Comment (alvarosanchez) [reply CONTRIBUTING.md:27]: Well, the problem described by #130 still persists when running with `java -jar` without arguments

Comment (jameskleeh) [reply CONTRIBUTING.md:27]: I've just pushed the fix for that problem, so this line can be removed

**Comment by jameskleeh** [comment]
Looks great - please change Category to be a normal class with static strings and change the API to return a String. This is a more extensible approach and we aren't doing any comparisons of categories that would make an enum valuable.

**Comment by alvarosanchez** [comment]
> Looks great - please change Category to be a normal class with static strings and change the API to return a String. That is a more extensible approach and we aren't doing any comparisons of categories that would make an enum valuable.

Done.

**Pull request merged**